.bit/responses/2.7-Week 2 Step 7.md

```diff
@@ -1,121 +1,55 @@
-        }
-    })
-
-    let data = await resp.json();
-    
-    return data; 
-}
-```
-
-Finally, we have to specify the request headers. Go back to the Face API documentation [here](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395236), and find the **Request headers** section. 
-
-There are two headers that you need. I've provided the format below. Enter in the two header names and their two corresponding values. 
-
-FYI: The `Content-Type`  header should be set to`'application/octet-stream'`.  This specifies a binary file.
-
-```js
-    //COMPLETE THE CODE
-    let resp = await fetch(uriBase + '?' + params.toString(), {
-        method: '<METHOD>',  //WHAT TYPE OF REQUEST?
-        body: '<BODY>',  //WHAT ARE WE SENDING TO THE API?
-      
-      	//ADD YOUR TWO HEADERS HERE
-        headers: {
-            '<HEADER NAME>': '<HEADER VALUE>'
-        }
-    })
-```
-
-Lastly, we want to call the `analyzeImage` function in `module.exports`. Add the code below into `module.exports`.
-
-Remember that `parts` represents the parsed multipart form data. It is an array of parts, each one described by a filename, a type and a data. Since we only sent one file, it is stored in index 0, and we want the `data`  property to access the binary file– hence `parts[0].data`. Then in the HTTP response of our Azure function, we store the result of the API call.
-
-```js
-//module.exports function
-
-//analyze the image
-var result = await analyzeImage(parts[0].data);
-
-context.res = {
-	body: {
-		result
-	}
-};
-
-console.log(result)
-context.done(); 
-
-```
-
-**To move on, commit the code from your function to the repository. Once your function works, you will be able to move on!**
+---
+files: n/a
+stepType: IssueComment
+scripts: n/a
+week: 2
+step: 7
+name: Week 2 Step 7
+---
+
+
+## Week 2 Step 7
+
+### Setting up the Twilio API
+Twilio is a cloud communication platform used to **programmatically** send SMS messages, calls, etc. You will be using the Twillio API to send yourself information from the Face data you obtained in the last issue.
+
+### 📝 Task 1: Set up your Twilio Account with a Dedicated Number
+
+<details>
+<summary>❓Where do I sign up?</summary>
+</br>
+* Go to the twilio website: https://www.twilio.com/try-twilio, and create an account here.
+</details>
+</br>
+
+Don't worry about paying for anything, you will be receiving **10 dollars of credit from Twilio**, which is more than enough for this project. Numbers should only cost 1 dollar.
+
+<details>
+<summary>❓How do I set up a new number?</summary>
+</br>
+Navigate to your dashboard, then press the three dots on the navigation panel to the left of your screen. Click on the 'phone numbers' option, then press the blue button in the top right corner to buy your own number, preferably from your location. 
+</details>
+</br>
+
+### 📝 Task 2: Set up your local environment
```

Comment:
**CHANGE THIS**: The course is structured so that each step *only has one task*. Students complete this task to **move on** within the course. To "move on," students can either (you choose which one):
* Make a PR with the correct files
* Submit code that runs correctly
* Comments something

Basically, just convert your "tasks" into sections instead, and have one overall task.